src/app.jsx

```diff
@@ -439,12 +439,14 @@ const ForceDeleteModal = ({ selected, itemPath, errorMessage }) => {
 
     const forceDelete = () => {
         cockpit.spawn(["rm", "-rf", itemPath], { superuser: "try" })
-                .then(Dialogs.close, (err) => { Dialogs.show(<DeleteErrorModal errorMessage={err.message} />) });
+                .then(Dialogs.close, (err) => { Dialogs.show(<ForceDeleteModal selected={undefined} itemPath={undefined} errorMessage={err.message} />) });
     };
 
-    const modalTitle = selected.type === "file"
-        ? cockpit.format(_("Force delete file $0?"), selected.name)
-        : cockpit.format(_("Force delete directory $0?"), selected.name);
+    const modalTitle = selected
```

Comment:
Just use the same title - no need title in case force deletion failed. So just remove the 'Unable To Delete' title.